joomla/joomla-cms
# Adds language flags to listing tables

#### Description

This PR adds language flags to listing tables in menu items, categories, articles, featured articles, modules, banners, news feeds, tags and contacts (and their modals) in administrator panel.

Example:
![langs](https://cloud.githubusercontent.com/assets/9630530/12054484/f31d8314-af1b-11e5-8149-34047677a8d8.png)
#### How to test
1. Fresh Joomla install with multilanguage (several languages).
2. Apply this patch
3. Check if flags are displayed in menu items, categories, articles, featured articles, modules, banners, news feeds, tags and contacts (and their modals) listings.
4. Do the same for hathor template.

For com_weblinks check https://github.com/joomla-extensions/weblinks/pull/176
#### Observations

Suggestions are welcome.
